js/ToDoList.js

```diff
@@ -25,7 +45,27 @@ class ToDoList {
     const ullist = document.getElementById("conteaner-list");
     ullist.innerHTML = "";
 
-    this.tasks.forEach((task, index) => {
+    var listFilterTasks = [];
+    if (this.filterTask == "Low") {
+      listFilterTasks = this.tasks.filter(
+        (task) => task.priority === this.filterTask
+      );
+      console.log(listFilterTasks);
+    } else if (this.filterTask == "high") {
+      listFilterTasks = this.tasks.filter(
+        (task) => task.priority === this.filterTask
+      );
+      console.log(listFilterTasks);
+    } else if (this.filterTask == "Medeum") {
+      listFilterTasks = this.tasks.filter(
+        (task) => task.priority === this.filterTask
+      );
+      console.log(listFilterTasks);
+    } else {
+      listFilterTasks = this.tasks;
+    }
+
+    listFilterTasks.forEach((task, index) => {
       const listItem = document.createElement("li");
```

Comment:
here you can add the if condition to check the filter value to determine whether to render the task or not based on its priority